lib/tasks/thor/digest.rb

```diff
@@ -0,0 +1,17 @@
+class DradisTasks < Thor
+  class Digest < Thor
+    namespace 'dradis:digests'
+
+    desc 'send_digests', 'Send a daily digest to all users'
+    def send_digests
```

Comment:
see my DigestSender comment, we may want to include this in the renaming effort.